### DISCUSSION TITLE
Add SSH support (like SFTP)

### DISCUSSION THREAD

**Discussion by strich3** [pelican-dev/panel#551] [Ideas]
Why would this be useful?

You do not have to open a website to just type in a quick command, also the SSH Terminal is just a niche thing to have.

**Comment by gOOvER** [comment]
I' m faster with putty. I think, thats not needed

**Reply by strich3** [reply]
What about autocompletion?

**Comment by notAreYouScared** [comment]
We by no means want to have this type of responsibility. There are other tools that are made for that.

**Reply by strich3** [reply]
What do you mean by that?

**Reply by notAreYouScared** [reply]
You're asking for a ssh console that's for the machine as a whole? If so, all it would take is someone's admin account to get compromised and they now have access.

**Reply by strich3** [reply]
No, not at all. I‘m talking about gameservers, like SFTP already exists

**Reply by notAreYouScared** [reply]
We already have a console for the docker container... that is the gameserver...

**Reply by strich3** [reply]
yes, but to my knowledge you can only access it through the panel website and it doesn‘t have autocompletion which is for me a very important feature. And lastly it isn‘t resizeable. SSH would maybe fix all the problems

**Reply by notAreYouScared** [reply]
We can look into making it popup, but it's going to retain its current functionality.

**Reply by strich3** [reply]
So autocompletion will not come?

**Reply by notAreYouScared** [reply]
> So autocompletion will not come?

Probaly not. We can look into after we have a stable beta
